doc/user_guide/run.rst

```diff
@@ -7,9 +7,10 @@ Invoking Pylint
 
 Pylint is meant to be called from the command line. The usage is ::
 
-   pylint [options] module_or_package
+   pylint [options] modules_or_packages
```

Comment:
Just an idea - but maybe we should indicate that we accept multiple arguments in usage section?

Something along a lines of:
- `pylint [options] module_or_package...`
- `pylint [options] module_or_package [module_or_package...]`
- other format?